modules/imaging_qc/README.md

```diff
@@ -0,0 +1,34 @@
+## Imaging Quality Control Module
+
+#### Purpose
+The purpose of this module is to verify and help ensure imaging data has been uploaded into LORIS and reviewed for Quality Control (QC). It provides a queryable interface to check at a glance the completeness of imaging data collection for a study, and the status of QC review processes on them.  
+
+Note: this module is not the module in which visual/radiological QC is performed on scans, just the module in which QC assessments are *reviewed*. Links are included in this module so the user can jump to the Imaging Browser to mark QC assessments on images and visits.
+
+#### Intended Users
+This module is used by study administrators, imaging scientists, analysts, coordinators, and clinicians.
+
+#### Scope
+The Imaging QC module allows you to view specific data on uploaded image files, with many additional filtering options. The resulting table has a column for QC status, so you can see whether the upload has passed QC or not. The table provides a link to each file. You can download the table in `.csv` format. 
+
+Note: To QC a scan, the user must click the link leading to the Imaging Browser, and enter the QC details in that module.
+
+#### Configuration caveat 
+
+This module has a key dependency - the `mri_parameter_form` instrument must installed. 
+If not, this module will not load and an error message will indicate that `Table 'LORIS.mri_parameter_form` is missing. 
+A sample MRI Parameter Form is available with our sample dataset in this repo: `raisinbread/instruments/NDB_BVL_Instrument_mri_parameter_form.class.inc`.  Please customize this PHP instrument for your project, and install it following the instructions in our Setup Guide for behavioural instruments, so that it appears among the database tables. Also consult the Setup Guide for recommended steps to configure and populate this instrument for your battery. 
+
+#### Permissions
+The `quality_control` permission gives users access to this module and its features.
+
+#### Interactions With LORIS
+
+##### Configuration
```

Comment:
1. "Configuration" header is a parallel level to "Interactions" in the other READMEs, not a subheading. (I can't remember which comes first, but you can check another module for a reference.
2. The other READMEs don't have subheadings for each module, they only have bullet points. This should be consistent.

Reply:
thanks dave. I fixed it so that the three module names under Interactions with LORIS are shown as bullets, not other subheadings. For number 1, I think you were referring to the Configuration module heading, which I changed to bullets. The actual "Configuration" header is on a parallel level to Interactions.